### DISCUSSION TITLE
DeepMD raw file convert to dataset.xyz for asap

### DISCUSSION THREAD

**Discussion by Wangmumu0403** [deepmodeling/deepmd-kit#1775] [Q&A]
Now I  have prepare this raw file for train data, I want to distinguish this structure by using asap and using energy , so. I want to ask   do we have module for raw file to convert the data (coord.raw and box.raw )directly to dataset.xyz for asap ?
 
![Selection_719](https://user-images.githubusercontent.com/95537862/174615405-91324388-2442-443d-ae58-80dd395aa69e.png)

>

**Comment by njzjz** [comment]
Please use [dpdata](https://github.com/deepmodeling/dpdata)
